src/sharepoint/provisioning/objecthandlers/objectcomposedlook/objectcomposedlook.ts

```diff
@@ -1,7 +1,9 @@
 "use strict";
 
 /// <reference path="..\schema\icomposedlook.d.ts" />
-import { replaceUrlTokens, getRelativeUrl } from "../../../Util";
+/// <reference path="../../../utils/util" />
```

Comment:
Obviously this is redudant. Waiting to see what the group decides the "right" way to do it is.